### PR TITLE
Test response code from a bad token

### LINKED ISSUE #31: Add test cases for failures
We should have test cases covering
- HTTP failure, like a 500 with gibberish content
- Request failure, like a 403 with a JSON message property

### PR DESCRIPTION
**Pull request by tmcw**
Fixes #31 cc @1ec5 for the review

### CODE REVIEW

File: MapboxDirections.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -632,6 +635,7 @@
 				39D04EFCE1A7300374AA5FBF /* Frameworks */,
 				CA9D333DD4D637ED30DA1B5D /* [CP] Embed Pods Frameworks */,
 				B146922AAE71DC82A56BBA60 /* [CP] Copy Pods Resources */,
+				43F82C8111AB3A1D45F1197F /* Embed Pods Frameworks */,
```
Comment (1ec5) [inline MapboxDirections.xcodeproj/project.pbxproj]: That’s weird, there’s a duplicate Embed Pods Frameworks target now. Can you make sure you’re on the latest CocoaPods beta and try again? (The 1.1.0 betas add support for Swift 2.3 and probably account for the [CP] steps above.)

Comment (tmcw) [inline MapboxDirections.xcodeproj/project.pbxproj]: I updated CocoaPods to 1.0.1 and rebuilt. There's still a lot of change to the pbxproj.

Comment (1ec5) [inline MapboxDirections.xcodeproj/project.pbxproj]: You’d have to either `pod deintegrate` or delete all the CocoaPods-generated phases first, then `pod install` again. Or, if either of these “Embed Pods Frameworks” phases is empty, you can safely remove it.

**Comment by tmcw** [comment]
Closing, stale.

**Pull request closed without merging**